src/test/regress/expected/aggregate_support.out

```diff
@@ -432,6 +432,21 @@ select key, count(distinct aggdata)
 from aggdata group by key order by 1, 2;
 ERROR:  type "aggregate_support.aggdata" does not exist
 CONTEXT:  while executing command on localhost:xxxxx
+-- GROUPING parses to GroupingFunc, distinct from Aggref
+-- These three queries represent edge cases implementation would have to consider
+-- For now we error out of all three
+select grouping(id)
+from aggdata group by id order by 1 limit 3;
+ERROR:  could not run distributed query with GROUPING
+HINT:  Consider using an equality filter on the distributed table's partition column.
```

Comment:
If I do a quick test we actually do correctly send the grouping to the workers. As you describe in the PR it will always return `0` as we do not support ROLLUPS etc. But I feel we don't have to artificially prevent the `grouping` syntax in the targetlist given it seems to work for me.

What is your reasoning behind specifically throwing an error?

Reply:
It raises an error message still when `pullUpIntermediateRows` is true

Reply:
nvm: read the last paragraph of the description just now, you can ignore this part